.github/workflows/main.yml

```diff
@@ -1,5 +1,10 @@
 name: Build
-on: [push]
+
+on:
+  push:
+    branches: [main]
```

Comment:
Would `on [push, pull_request]` be enough here? Not sure we need to restrict things to `main` do we?

Reply:
Indeed, we probably don't need that. Corrected.